synapseclient/models/services/storable_entity.py

```diff
@@ -1,103 +1,162 @@
 """Script used to store an entity to Synapse."""
 
-from typing import TYPE_CHECKING, Dict, Optional, Union
+from typing import TYPE_CHECKING, Any, Dict, Optional, Union
 
 from opentelemetry import trace
 
 from synapseclient import Synapse
 from synapseclient.api import (
     create_access_requirements_if_none,
-    post_entity,
-    put_entity,
+    get_entity_id_bundle2,
+    post_entity_bundle2_create,
+    put_entity_id_bundle2,
 )
 from synapseclient.core.utils import get_properties
 
 if TYPE_CHECKING:
-    from synapseclient.models import File, Folder, Project
+    from synapseclient.models import (
+        Annotations,
+        Dataset,
+        EntityView,
+        File,
+        Folder,
+        Project,
+        Table,
+    )
 
 
 async def store_entity(
```

Comment:
This function is incomplete, more work to support all functionality from the existing synapse class `.store()` function is needed